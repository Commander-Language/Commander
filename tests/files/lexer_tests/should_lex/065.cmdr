// Test that line endings for multi-line strings work for LF line endings
a = 'Hi \
there \
bob';