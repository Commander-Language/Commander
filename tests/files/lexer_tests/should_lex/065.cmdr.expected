VARIABLE 'a' 2:1
EQUALS '=' 2:3
STRINGVAL ''' 2:5
STRINGLITERAL 'Hi there bob' 2:6
ENDSTRINGVAL ''' 4:4
SEMICOLON ';' 4:5
END_OF_FILE 4:6
